src/components/Timer/index.tsx

```diff
@@ -47,48 +55,40 @@ export default function Question({
     setRemainingTime(duration);
   }, [selectedQuestionId, duration]);
 
+  function formatTimeLeft(time) {
+    const minutes = Math.floor(time / 60);
+    let seconds = `${time % 60}`;
+
+    if (parseInt(seconds) < 10) {
+      seconds = `0${seconds}`;
+    }
+
+    return `${minutes}:${seconds}`;
+  }
+
   return (
-    <div className={classNames(styles.timer, { [styles.running]: running })}>
-      {!!remainingTime && (
-        <Button
-          className={classNames('absolute', styles.playBtn)}
-          aria-label="start / pause timer"
-          aria-pressed={running}
-          icon={<Icon name={running ? 'pause' : 'play'} />}
-          onClick={() => setIsRunning(!running)}
-          basic
-        />
-      )}
-      {!!remainingTime && remainingTime > 0 ? (
-        <div className="bold">{title}</div>
-      ) : (
-        <div>
-          <div className="bold">Time is up</div>
-          <div>Submitting in {remainingTime + 3}s</div>
-        </div>
-      )}
-      {!!remainingTime && remainingTime > 0 && (
-        <div className="time flex mt-md justifyCenter">
-          <div className="min">
-            {remainingTime / 60 < 10 ? '0' : ''}
-            {Math.floor(remainingTime / 60)}
-          </div>
-          <div className={classNames('mx-md', { [styles.blinking]: running })}>:</div>
-          <div className="sec">
-            {remainingTime % 60 < 10 ? '0' : ''}
-            {remainingTime % 60}
-          </div>
-        </div>
-      )}
-      {!!remainingTime && (
-        <div
-          className={styles.timerIndicator}
-          style={{
-            top: ((duration - remainingTime) * 100) / duration + '%',
-            height: (remainingTime * 100) / duration + '%',
-          }}
-        />
-      )}
+    <div className={styles.baseTimer}>
+      <svg className={styles.baseTimerSvg} viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
+        <g className={styles.baseTimerCircle}>
+          <circle className={styles.baseTimerPathElapsed} cx="50" cy="50" r="45" />
+          <path
+            id="base-timer-path-remaining"
+            stroke-dasharray={circleDasharray}
+            className={classNames({
```

Comment:
Can we use some library for this rather than doing manually if some nice library is available?

Reply:
we are already using `classnames` library

Reply:
I was talking about this whole timer svg component. there should be some library which does this?

Reply:
it is fine.  we don't need any library just for a timer. We can easily create it from scratch and use the animation which we want. here is the link 
https://css-tricks.com/how-to-create-an-animated-countdown-timer-with-html-css-and-javascript/

Reply:
Can you add this link in the component as a comment somewhere?